cmd/cstor-pool-mgmt/crdops/common.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2018 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package crdops
+
+const (
+	// SuccessSynced is used as part of the Event 'reason' when a resource is synced
+	SuccessSynced = "Synced"
+	// ErrResourceExists is used as part of the Event 'reason' when a resource fails
+	// to sync due to a resource of the same name already existing.
+	ErrResourceExists = "ErrResourceExists"
+
+	// MessageResourceExists is the message used for Events when a resource
+	// fails to sync due to a resource already existing
+	MessageResourceExists = "Resource %q already exists and is not managed by resource"
```

Comment:
I am finding this message difficult to understand

Reply:
ok, will change